plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/table/actions/install.tsx

```diff
@@ -66,6 +71,10 @@ export default function Install( props: InstallProps ) {
 					);
 					stopInstall();
 				} );
+
+				recordEvent( 'marketplace_product_installed', {
```

Comment:
Not sure about installed and failed events. They are not much about user behavior and may not have a place in the Tracks system. 

Also more difficult to test the failed events. If we decide to keep them, I'll come up with a way